### DISCUSSION TITLE
Safari Crashing - Firefox is OK

### DISCUSSION THREAD

**Discussion by AbbevilleArt** [mscherotter/galeryst#4] [Q&A] [answered]
Hi

Am using Safari Version 15.0 (16612.1.29.41.4, 16612) - and unable to view any art or galleries, but the same pages are fine in Firefox, any ideas?

Thanks

**Comment by mscherotter** [comment, marked as answer]
There is an update to Safari version 15.1 that fixes this problem. (16612.2.9.1.30, 16612)

**Comment by AbbevilleArt** [comment]
Hi

Thats an interesting reply

It’s a rare thing in the Mac world to be told that I need to update my entire OS - an unproven OS - Monterey is less than 2 months old - in order to run browser based software.

It excludes all Mac’s with Nvidia GPU

It excludes iMac Pro’s from 2017 onwards

More importantly, I’m not willing to put myself in the position to tell my clients that they need a major upgrade - an OS upgrade is not a trivial matter - to view a website.

Best of luck

Mark

Phone -  WhatsApp - 07703 697983
Workshop - https://www.42abbeville.com
Design and Make - https://www.victoriabarker.london

**Comment by mscherotter** [comment]
From what I heard from another Galeryst user, it is just an update to Safari, not the entire OS.
Michael

**Comment by mscherotter** [comment]
https://www.macrumors.com/2021/10/27/apple-releases-safari-15-1-macos-big-sur-catalina/

**Comment by AbbevilleArt** [comment]
Thanks

My mistake

Am downloading 15.2 now, will give you a report

Best

Mark

Phone -  WhatsApp - 07703 697983
Workshop - https://www.42abbeville.com
Design and Make - https://www.victoriabarker.london

**Comment by AbbevilleArt** [comment]
K - 15.2 loads the gallery and foyer - Thanks

Question - when I select a piece, there is an orange button with a white cross - when I hit this it brings me out to the main page as opposed to back to the wing itself - is this deliberate?

Sorry to be a pain - I love the concept and would love to be able to use it professionally

Mark

Phone -  WhatsApp - 07703 697983
Workshop - https://www.42abbeville.com
Design and Make - https://www.victoriabarker.london

**Comment by mscherotter** [comment]
When you click on a piece a first time, it animates to be in front of it.
Clicking on the piece a second time makes it full screen.
Clicking on a third time goes back to walking around the gallery.
The x closes the 3D view.
Michael

**Comment by AbbevilleArt** [comment]
:) Not in Safari - sorry


Phone -  WhatsApp - 07703 697983
Workshop - https://www.42abbeville.com
Design and Make - https://www.victoriabarker.london
